CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 * Pairing with pincode
 * Handle os.remove failing
 * Fix disconnecting NMDevice
-* Untranslated string in DiscvManager
+* Untranslated strings (@cwenling / Colomban Wendling)
```

Comment:
Heh, thanks for the cred :)  FWIW my GitHub username lacks a `d` here though :wink:

Reply:
Oh gosh. :see_no_evil: #1369

Reply:
don't sweat it either ^^